hubV2/storage/provider/SPECS.MD

```diff
@@ -0,0 +1,26 @@
+#Storage Providers
+
+Each storage provider must add these API/methods:
+
+
+
+```python
+
+
+class StorageProvider():
```

Comment:
This looks a lot like a base class. Why not just implement it as a base class?

Reply:
Good suggestion. Will do that.

Reply:
I think they are actually the same.

<Hub format> -> ADAPTOR <- <SOme External tool like annotation, visualization etc.>

Do you think .torch() & tensorflow() goes in here as well? RIght now the plan was to put them in integrations.

So Integration is for model and adaptors are for tools like annotations etc.

Reply:
i don't see why `.torch()`/`.tensorflow()` should be implemented in a new directory.

`ds.torch()` <-- yields the torch **dataset**. so it should be defined on `hub.Dataset`. same with `.tensorflow()`

also, `tensor.torch()` <-- yields a torch **tensor**. so this should be a `hub.Tensor` method.